src/language/lexer.js

```diff
@@ -668,8 +668,22 @@ function readBlockString(source, start, line, col, prev): Token {
       );
     }
 
-    // Escape Triple-Quote (\""")
-    if (
+    if (code === 10) {
+      // new line
+      ++position;
+      ++lexer.line;
+      lexer.lineStart = position;
+    } else if (code === 13) {
+      // carriage return
```

Comment:
@langpavel Can you please add test cases that cover `\r\n` and `\n\r`?

Reply:
Added